src/js/view/controls/components/menu/menu.js

```diff
@@ -22,6 +22,7 @@ export default class Menu extends Events {
         this.toggle = this.toggle.bind(this);
         this.name = _name;
         this.title = _title || _name;
+        this.localization = _localization;
```

Comment:
We should avoid storing this in a property if possible. It's a pretty large JSON file (all of our localized copy), and if it's stored as a property it won't be garbage collected at the end of construction.

Reply:
Great catch. I'll update this to be just `_localization.prev` as that's all that this class ever needs to access.